css/style.css

```diff
@@ -98,6 +98,41 @@ body {
   margin: 0 auto;
 }
 
+.card-component-container {
+  max-width: 80%;
```

Comment:
You don't need max-width here.

Reply:
I don't think so.... you'll see when I finish styling, but I'll consider it